vm/actor/src/builtin/errors.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::fmt;
+
+#[derive(Debug, PartialEq)]
+pub enum Error {
+    /// Error orginating from state
+    State(String),
+    /// Error originating from encoding arbitrary data
+    Encoding(String),
+}
+
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Error::State(msg) => write!(f, "{}", msg),
+            Error::Encoding(msg) => write!(f, "{}", msg),
+        }
+    }
+}
```

Comment:
This is never used. Wouldn't make sense to have a builtin actor specific errors anyway

I think for error handling in cases like this where it would just be forwarding the error as a wrapped string, probably makes sense to just return a `String` as the error return, as it shouldn't matter what variant the error was, you would just need a string to log to console explaining.